internal/semantic/parse_test.go

```diff
@@ -14,8 +14,7 @@ func TestParse(t *testing.T) {
 
 	ecosystems := lockfile.KnownEcosystems()
 
-	// todo: remove once CRAN is supported by lockfile
-	ecosystems = append(ecosystems, "CRAN")
+	ecosystems = append(ecosystems, "Alpine", "Debian", "Ubuntu")
```

Comment:
should these be consts defined in `pkg/lockfile`?

Reply:
Given a big part of v2 is about sorting our "ecosystem" type, I don't think we should be making any constants right now 😅

(I believe in the long-run these should eventually be sourced from the upcoming `osvschema` package)

Reply:
I think we have the constants already defined for example https://github.com/google/osv-scanner/blob/main/pkg/lockfile/apk-installed.go#L11?

Reply:
but it doesn't have one for `Ubuntu`, and `lockfile` won't be available when this is moved to `scalibr` - I can change if you really want, I just don't think it's a big deal for these two lines to be using the string format given the related upcoming movement that'll likely end up changing this anyway 🤷

Reply:
fair - I think the point is to avoid having free-text ecosystem strings and the ecosystems should be from [osv-schema](https://github.com/ossf/osv-schema/blob/main/bindings/go/osvschema/constants.go).